app/Console/Commands/RosterUpdate.php

```diff
@@ -83,6 +83,22 @@ public function handle() {
             $user->save();
         }
 
+        $visitors = User::where('visitor', 1)->where('status', 1)->where('api_exempt', 0)->get();
+        foreach ($visitors as $visitor) {
+            $res = $client->request('GET', Config::get('vatusa.base').'/v2/user/'.$visitor->id, ['http_errors' => false]);
+            if ($res->getStatusCode() == "200") {
+                $v = json_decode($res->getBody());
+                $visitor->fname = $v->data->fname;
+                $visitor->lname = $v->data->lname;
+                $visitor->email = $v->data->email;
+                $visitor->rating_id = $v->data->rating;
+                $visitor->visitor_from = $v->data->facility;
+                $visitor->save();
+            } else {
+                exit(1);
```

Comment:
Do we want to exit here or try again/just continue? If there is a random error I'd rather it handle that gracefully and continue and miss 1 person rather than a whole chunk. Then ideally that 1 person would be added the following hour

Reply:
@iccowan Thank you - that's a great point and I've changed it from an exit() to a continue.